kalite/testing/testrunner.py

```diff
@@ -111,6 +115,15 @@ def __init__(self, *args, **kwargs):
         self._bdd_only = kwargs["bdd_only"]  # Extra options from our custom test management command are passed into
         self._no_bdd = kwargs['no_bdd']      # the constructor, but not the build_suite function where we need them.
 
+        # Django < 1.7 serves static files using the staticfiles app, not from static root.
+        # This causes django_js_reverse not to get served to the client, so we manually copy it into distributed.
+
+        call_command("collectstatic_js_reverse", interactive=False)
+
+        ensure_dir(os.path.join(os.path.dirname(os.path.dirname(__file__)), "distributed", "static", "django_js_reverse", "js"))
```

Comment:
Should this be:
os.path.dirname(\_\_file\_\_)), "distributed", "static", __"js"__, "django_js_reverse", "js"
?